RealtimeCSG/ProjectSettings/ProjectVersion.txt

```diff
@@ -1 +1,2 @@
-m_EditorVersion: 5.6.4f1
+m_EditorVersion: 2021.2.7f1
+m_EditorVersionWithRevision: 2021.2.7f1 (6bd9e232123f)
```

Comment:
Can you revert this?
I'm fairly sure there are people who are using realtime-csg who are on older versions than 2021
upgrading is easier than downgrading.
Better yet, just delete the file

Reply:
This would be the last issue

Reply:
Done.